bot.py

```
@@ -253,6 +256,45 @@ async def voice_to_text(message: types.Message, bot: Bot):
             os.remove(wav_path)
 
 
+@router.inline_query()
+async def inline_emojis(inline_query: types.InlineQuery, bot: Bot):
```

Ruff (0.12.2)

Unused function argument: `bot`

(ARG001)